common/lib/capa/capa/inputtypes.py

```diff
@@ -316,8 +317,12 @@ def _get_render_context(self):
             'value': self.value,
             'status': Status(self.status, self.capa_system.i18n.ugettext),
             'msg': self.msg,
-            'STATIC_URL': self.capa_system.STATIC_URL,
+            'STATIC_URL': self.capa_system.STATIC_URL
         }
+
+        if self.response_data and self.response_data['label'] != '':
+            context['response_data'] = self.response_data
+
```

Comment:
Note: This check is just for the spike and temporary, and will be removed when we will do for all capa problems.